src/sas/qtgui/MainWindow/GuiManager.py

```diff
@@ -423,16 +424,18 @@ def perspectiveChanged(self, new_perspective_name: str):
         #
         # Selection on perspective choice menu
         #
-        if isinstance(new_perspective, FittingWindow):
+        # checking `isinstance`` fails in PySide6 with
```

Comment:
Hmmmm, it's clearly to do with FittingWindow inheriting from ABC ulitimately, I think it might be to do with the metaclass (or lack thereof) in Perspecive.